### PR TITLE
arch & colors changes

### PR DESCRIPTION
**Pull request by mohamedrefaysayed**
- Change App Icon
- Change App Colors
- Change app Font "SpaceGrotesk"

[
Screenshot 2024-04-06 061423
WhatsApp Image 2024-04-06 at 06 14 44_78d2181f
](url)

New Font
WhatsApp Image 2024-04-07 at 03 07 08_ee8d9ed1

### CODE REVIEW

File: lib/core/themes/app_colors.dart [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/material.dart';
 
 class AppColors {
-  static const Color mainPurple = Color(0xffc82bce);
-  static const Color deepPurple = Color(0xff5b1c67);
-  static const Color deepBlue = Color(0xff37145a);
+  static const Color kPurple = Color(0xffA64EDD);
```
Comment (omarahmedx14) [inline lib/core/themes/app_colors.dart]: please don't use k before variable naming.

File: lib/core/themes/text_styles.dart [hunk 2]
```diff
@@ -7,46 +7,55 @@ class TextStyles {
     fontSize: 10.sp,
     fontWeight: FontWeight.w700,
     color: AppColors.white,
+    fontFamily: 'SpaceGrotesk',
   );
   static TextStyle textstyle12 = TextStyle(
```
Comment (omarahmedx14) [inline lib/core/themes/text_styles.dart]: what's ( textstyle12 ) ?

it's better if we name it like ( font12White700w ) or ( font13DarkBlueMedium ) any similar descriptive name.

**Review by omarahmedx14 — CHANGES_REQUESTED** [review]
left some notes

File: lib/core/themes/app_colors.dart (hunk 1 above)
Comment (mohamedrefaysayed) [reply lib/core/themes/app_colors.dart]: done

File: lib/core/themes/text_styles.dart (hunk 2 above)
Comment (mohamedrefaysayed) [reply lib/core/themes/text_styles.dart]: done

**Review by omarahmedx14 — APPROVED** [review]

**Pull request merged**